app/src/main/java/com/google/android/samples/socialite/ui/home/timeline/TimelineViewModel.kt

```diff
@@ -47,11 +50,17 @@ class TimelineViewModel @Inject constructor(
 
     // Single player instance - in the future, we can implement a pool of players to improve
     // latency and allow for concurrent playback
-    var player by mutableStateOf<Player?>(null)
+    var player by mutableStateOf<ExoPlayer?>(null)
 
     // Width/Height ratio of the current media item, used to properly size the Surface
     var videoRatio by mutableStateOf<Float?>(null)
 
+    // Preload Manager for preloaded multiple videos
+    private val enablePreloadManager: Boolean = true
+    private lateinit var preloadManager: PreloadManagerWrapper
+
+    var timeToFirstFrame = 0L
```

Comment:
Can we rename this variable to `playbackStartTimeMs`?

I saw later you're calculating `timeToFirstFrame` in `onRenderedFirstFrame` as:

```
timeToFirstFrame = System.currentTimeMillis() - timeToFirstFrame
```

I think changing it to

```
val timeToFirstFrameMs = System.currentTimeMillis() - playbackStartTimeMs
```

will make it clear on how this metric is calculated for the developer who looks at the sample code.